src/Libt8.jl

```diff
@@ -7222,8104 +7243,9425 @@ function t8_element_shape_compare(element_shape1, element_shape2)
+| last\\_local\\_tree     | The global index of the last local tree on this process. -1 if this processor is empty.                                                                                                                                                                                                        |
+| global\\_num\\_trees    | The total number of global trees                                                                                                                                                                                                                                                               |
+| ghosts                  | If not NULL, the ghost elements.  # See also [`t8_forest_ghost`](@ref).h                                                                                                                                                                                                                       |
+| element\\_offsets       | If partitioned, for each process the global index of its first element. Since it is memory consuming, it is usually only constructed when needed and otherwise unallocated.                                                                                                                    |
+| global\\_first\\_desc   | If partitioned, for each process the linear id (at maxlevel) of its first element's first descendant. t8_element_set_linear_id. Stores 0 for empty processes. Since it is memory consuming, it is usually only constructed when needed and otherwise unallocated.                              |
+| tree\\_offsets          | If partitioned for each process the global index of its first local tree or -(first local tree) - 1 if the first tree on that process is shared. Since this is memory consuming we only construct it when needed. This array follows the same logic as *tree_offsets* in [`t8_cmesh_t`](@ref)  |
+| local\\_num\\_elements  | Number of elements on this processor.                                                                                                                                                                                                                                                          |
+| global\\_num\\_elements | Number of elements on all processors.                                                                                                                                                                                                                                                          |
+| profile                 | If not NULL, runtimes and statistics about forest\\_commit are stored here.                                                                                                                                                                                                                    |
+"""
+# struct t8_forest
```

Comment:
If I understand correctly, you commented this and the following lines by hand? Is that correct and if yes, why was that necessary?

Reply:
There is a circular dependency with `t8_forest_t` which throws an error. Since this struct is not supposed to be read and modified directly anyway, I commented this section out and made the struct opaque. I added a comment for clarification.

Reply:
This is quick-fix/workaround. Since t8code and Julia is changing so fast I do not see any worth in spending time into a long term solution.

Reply:
Hm, this is not ideal since now this file is not the same as you get when you run the generator in `dev/`. Would it be hard to add this patch to `dev/fixes.sh`? Otherwise I see a good chance that we will miss this the next time we run the generator again.

Reply:
I know and agree. I added an output in `fixes.sh` for the user to apply the fix manually. I won't invest time to cook up a brittle automatic solution which most probably will break and/or gets obsolete in the near future anyway.

Relying on a `fixes.sh` is a nuisance anyway ...

Reply:
Yeah, that's true. Thanks! Having in mind that you need this PR to be merged to continue working on the other PR(s) in Trixi.jl, I'm fine with the current workaround. But it would be nice to find a proper fix in the future.